tests/MapperTestUtility.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
```

Comment:
Sorry I forgot, actually I copied the function and this part was default in all the pages. I will change it.